spring-cloud-kubernetes-client-discovery/src/main/java/org/springframework/cloud/kubernetes/client/discovery/KubernetesInformerDiscoveryClient.java

```diff
@@ -227,42 +227,40 @@ private int findEndpointPort(List<CoreV1EndpointPort> endpointPorts, String prim
 
 	@Override
 	public List<String> getServices() {
-		List<V1Service> services = this.properties.allNamespaces() ? this.serviceLister.list()
-				: this.serviceLister.namespace(this.namespace).list();
+		List<V1Service> services = properties.allNamespaces() ? serviceLister.list()
+				: serviceLister.namespace(namespace).list();
 		return services.stream().filter(this::matchServiceLabels).map(s -> s.getMetadata().getName())
 				.collect(Collectors.toList());
 	}
 
 	@PostConstruct
 	public void afterPropertiesSet() {
-		this.sharedInformerFactory.startAllRegisteredInformers();
-		if (!Wait.poll(Duration.ofSeconds(1), Duration.ofSeconds(this.properties.cacheLoadingTimeoutSeconds()), () -> {
-			log.info("Waiting for the cache of informers to be fully loaded..");
-			return this.informersReadyFunc.get();
+		sharedInformerFactory.startAllRegisteredInformers();
+		if (!Wait.poll(Duration.ofSeconds(1), Duration.ofSeconds(properties.cacheLoadingTimeoutSeconds()), () -> {
+			LOG.info(() -> "Waiting for the cache of informers to be fully loaded..");
+			return informersReadyFunc.get();
 		})) {
-			if (this.properties.waitCacheReady()) {
+			if (properties.waitCacheReady()) {
 				throw new IllegalStateException(
 						"Timeout waiting for informers cache to be ready, is the kubernetes service up?");
 			}
 			else {
-				log.warn(
-						"Timeout waiting for informers cache to be ready, ignoring the failure because waitForInformerCacheReady property is false");
+				LOG.warn(
+					() -> "Timeout waiting for informers cache to be ready, ignoring the failure because waitForInformerCacheReady property is false");
 			}
 		}
-		log.info("Cache fully loaded (total " + serviceLister.list().size()
+		LOG.info(() -> "Cache fully loaded (total " + serviceLister.list().size()
 				+ " services) , discovery client is now available");
 	}
 
 	private boolean matchServiceLabels(V1Service service) {
-		if (log.isDebugEnabled()) {
```

Comment:
this changes simply drops `if (log.isDebugEnabled()) {` because we are using LogAccesor now